src/students.rb

```diff
@@ -1,6 +1,8 @@
 require_relative 'person'
 
 class Student < Person
+  attr_accessor :classroom
+
```

Comment:
- [ ] Kindly ensure that the `classroom` object can only retrieve the value of instance of the variable. Getters method only provide read access to the attributes of an object. ℹ️
```suggestion
  attr_reader :classroom
```

Reply:
Again, no clarity in the suggestion. You are talking about the classroom object when marking code from students object code